com.playeveryware.eos/Runtime/EOS_SDK/Generated/AntiCheatCommon/LogGameRoundStartOptions.cs

```diff
@@ -67,13 +73,22 @@ public uint RoundTimeSeconds
 			}
 		}
 
+		public AntiCheatCommonGameRoundCompetitionType CompetitionType
+		{
+			set
+			{
+				m_CompetitionType = value;
+			}
+		}
```

Comment:
This looks _really_ weird. Are we certain that it's not a flaw?

Reply:
I'm looking at the downloaded zip, and this is how it's set up. All of the properties here have public sets, but no visible gets. I think it's because of the needs of ISettable, or something.

Is there anything we must do to validate that this is sane?

Reply:
I wanna get your take on this before merging in the PR.

Reply:
We should make a task on this to investigate it, but not block the integration of the 1.16.4 sdk.